### PR TITLE
Added color constants and new Color constructor to use them

### PR DESCRIPTION
**Pull request by hummingburd**

### CODE REVIEW

File: csharp/src/sdk/Geometry/Color.cs [hunk 1]
```diff
@@ -6,7 +6,35 @@ namespace Hypar.Geometry
     /// Color represents an RGBA color.
     /// </summary>
     public class Color
-    {   
+    {
+        public static readonly float[] Aqua = { 0.3f, 0.7f, 0.7f, 0.0f };
```
Comment (ikeough) [inline csharp/src/sdk/Geometry/Color.cs]: I'd like to do this how `System.Drawing.Color` works in .net. Convert all of these to static properties like:
```c#
public static Color Red
{
  get{ return new Color(1.0f, 0.0f, 0.0f, 0.0f);}
}
```
The difference would be this:
as proposed:
```
var red = new Color(Color.Red);
```
updated:
```
var red = Color.Red
```

File: csharp/src/sdk/Geometry/Color.cs [hunk 2]
```diff
@@ -56,6 +84,33 @@ public Color(float red, float green, float blue, float alpha)
             this.Alpha = alpha;
         }
 
+        /// <summary>
+        /// Construct a color from an red, green, blue, and alpha value array.
+        /// </summary>
+        /// <param name="rgba">An array of the red, blue, green, and alpha color components between 0.0 and 1.0.</param>
+        public Color(float[] rgba)
```
Comment (ikeough) [inline csharp/src/sdk/Geometry/Color.cs]: Pending the changes above, this constructor is no longer needed.

File: csharp/src/sdk/Geometry/Color.cs (hunk 1 above)
Comment (hummingburd) [reply csharp/src/sdk/Geometry/Color.cs]: Ah okay. That's more elegant, thanks.

File: csharp/src/sdk/Geometry/Color.cs (hunk 2 above)
Comment (hummingburd) [reply csharp/src/sdk/Geometry/Color.cs]: Right, saw that, deleted.

**Pull request merged**